packages/contracts/contracts/governance/UpgradeGovernor.sol

```diff
@@ -120,6 +120,41 @@ contract UpgradeGovernor is
         return super.proposalThreshold();
     }
 
+    /**
+     * @notice Function to queue a proposal to the timelock.
+     *         Added protocol for using custom time-lock zero delay for urgent situations.
+     */
+    function queue(
+        address[] memory targets,
+        uint256[] memory values,
+        bytes[] memory calldatas,
+        bytes32 descriptionHash
+    ) public virtual override returns (uint256) {
+        uint256 proposalId = hashProposal(targets, values, calldatas, descriptionHash);
+
+        require(state(proposalId) == ProposalState.Succeeded, "UpgradeGovernor: proposal not successful");
+
+        uint256 delay = _timelock.getMinDelay();
```

Comment:
Unrelated to this file, could you please rename the constructor argument names of the `UpgradeGovernor` and `TimeLock` contracts in deploy config to the following? The current names don't seem to be intuitive.
```
votingDelayBlocks => governorVotingDelay
votingPeriodBlocks => governorVotingPeriod
proposalThresholdTokens => governorProposalThreshold
votesQuorumFractionPercent => governorQuorumFraction
proposalExecutingLatencySeconds => timeLockMinDelay
```

Reply:
Yes I do remember :) For the renaming, I'll proceed to the next PR